app/public/login/index.html

```diff
@@ -0,0 +1,118 @@
+          <input
+            type="password"
+            id="password"
+            name="password"
+            placeholder="Password"
+            required />
+        </div>
+        <button type="submit" id="submit">Login</button>
+        <button id="noAccountCreateAccount">Create Account</button>
+        <div id="error-message" style="color: red">ERROR SHOW HERE</div>
+      </form>
+    </div>
+
+    <!-- <button id="send">Send message to client</button> -->
+    <div class="create-container">
+      <h2>Create Account</h2>
+      <form id="createAccountForm">
+        <div class="input-group">
+          <label for="newUsername">Username</label>
+          <input
+            type="text"
+            id="newUsername"
+            name="newUsername"
+            placeholder="Username"
+            required />
+        </div>
+        <div class="input-group">
+          <label for="newPassword">Password</label>
+          <input
+            type="password"
+            id="newPassword"
+            name="newPassword"
+            placeholder="Enter your password"
+            onInput="checkPassword()"
+            required />
+          <div id="set">
+            <i id="see" onclick="see()" class="far fa-eye"></i>
+            <div id="count">Length : 0</div>
+          </div>
+          <div id="lengthGreatThanEight">
+            <i class="far fa-check-circle"></i>
+            <span> Length more than 8.</span>
+          </div>
+          <div id="containsNumber">
+            <i class="far fa-check-circle"></i>
+            <span> Contains numerical character.</span>
+          </div>
+          <div id="containsSpecialCharacter">
+            <i class="far fa-check-circle"></i>
+            <span>Contains special character /[!@#$%^&*(),.?":{}|</span>
+          </div>
+          <div id="containUppercaseLetter">
+            <i class="far fa-check-circle"></i>
+            <span>Contains uppercase character.</span>
+          </div>
+          <div id="containLowercaseLetter">
+            <i class="far fa-check-circle"></i>
+            <span>Contains lowcase character.</span>
+          </div>
+          <div id="notContainSpace">
+            <i class="far fa-check-circle"></i>
+            <span>Shouldn't contain spaces.</span>
+          </div>
+        </div>
+        <div class="input-group">
+          <label for="newPasswordConfirm">Confirm Password</label>
+          <input
+            type="password"
+            id="newPasswordConfirm"
+            name="newPasswordConfirm"
+            placeholder="Confirm Password"
+            onInput="cofirmNewPassword()"
+            required />
+        </div>
+        <div id="matchNewPassword">
+          <i class="far fa-check-circle"></i> <span> Match your password.</span>
+        </div>
+        <button type="submit" id="createAccount">Create Account</button>
+      </form>
+    </div>
+  </body>
+  <script src="./constants.js"></script>
```

Comment:
The relative file path to constants is actually "../constants.js".
But I don't think there's any constants in there that you need, so I would recommend deleting this line.